release-notes/aws/v9.0.1.md

```diff
@@ -0,0 +1,53 @@
+## :zap: Giant Swarm Release 9.0.1 for AWS is now active for you! :zap:
+
+This release includes some reliability and user experience improvements.
+
+Below, you can find more details on components that were changed with this release.
+
+### Kubernetes [v1.15.11](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.15.md#changelog-since-v11510)
+- Updated from v1.15.5.
+- Fixed a regression where the kubelet would fail to update the ready status of pods.
+- Various other improvements to the management of cloud and cluster resources.
+
+### aws-operator v5.5.1
+- Streamlined image templating for core components for quicker and easier releases in the future.
+- Enabled per-cluster configuration of kube-proxy's `conntrackMaxPerCore` parameter.
+
+### chart-operator [v0.12.1](https://github.com/giantswarm/chart-operator/releases/tag/v0.12.1)
+
+- Added chartmigration resource for migrating from chartconfig to chart CRs.
+- Updated release resource.
+  - Do not wait when installing or updating long running Helm releases.
+  - Use version field from chart CR to reduce number of HTTP requests to pull chart tarballs.
+  - Wait for deleted Helm release before removing finalizer. 
+- Updated status resource.
+  - Improve reason field in CR status when installing a chart fails.
+- Removed legacy chartconfig controller.
+
+### cluster-operator [v0.23.8](https://github.com/giantswarm/cluster-operator/releases/tag/v0.23.8)
+
+- Stop setting IC replicas count.
+- Re-enable proxy protocol by default for AWS clusters.
```

Comment:
```suggestion
```
This was broken and fixed in intermediary cluster-operator releases, both of which are included in this platform operator so there's no need to mention regression and fix in releases notes.